### PR TITLE
[RSDK-8562] make control freq configurable

### PR DESCRIPTION
**Pull request by JohnN193**
https://viam.atlassian.net/browse/RSDK-8562

last week we realized that some controllers we use have poor performance at low velocities. Adjusting the control loop frequency is another way we can improve controller behavior. 

default frequency, set velocity 50 mm/s
image

high frequency, same PID
run582-uhigh-freq-performance

### CODE REVIEW

**Comment by martha-johnston** [comment]
nice job finding this fix! I think if we want to add frequency to the config there needs to be a scope doc?

**Comment by JohnN193** [comment]
i wrote a scope [here](https://docs.google.com/document/d/1sq5laDdOMaxNGuixMihmlIDKq09Ng1MAsz8hPTKUVnE/edit) but rand was saying i should just do the backend anyways. not sure how to go about submitting this scope tho

**Comment by martha-johnston** [comment]
I see. I assume don't merge until the scope is approved? but idk if Rand meant something else

**Comment by martha-johnston** [comment]
> not sure how to go about submitting this scope tho

I think you can just add it to a document ticket. maybe technically supposed to be part of an epic but this is so minor that seems unnecessary?

**Review by martha-johnston — APPROVED** [review]
lgtm pending scope approval

**Pull request merged**
